src/main/java/it/gov/pagopa/reward/dto/mapper/SynchronousTransactionRequestDTOt2TrxDtoOrResponseMapper.java

```diff
@@ -67,6 +67,9 @@ public SynchronousTransactionResponseDTO apply(SynchronousTransactionRequestDTO
         out.setChannel(request.getChannel());
         out.setInitiativeId(initiativeId);
         out.setUserId(request.getUserId());
+        out.setOperationType(request.getOperationType());
+        out.setAmount(request.getAmountCents());
```

Comment:
let's return amount not amountCents

every currency in other fields returned are euro (effectiveAmount, reward.accruedReward)
```suggestion
        out.setAmount(request.getAmount());
```